README.md

```diff
@@ -1,2 +1,35 @@
 [![Build Status](https://travis-ci.com/elegant-bro/arrayee.svg?branch=master)](https://travis-ci.com/elegant-bro/arrayee)
 # Work with arrays in elegant way
+
+## For contributors 
```

Comment:
maybe you can wrap all these commands into Makefile?

Reply:
@tigokr can we use composer commands? We are php developers :) 

https://getcomposer.org/doc/articles/scripts.md

`composer unit-tests` - like this

Reply:
I think we should not depends on composer so deep

Reply:
great idea =)

Reply:
ok... lets see how it will look like